tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -4882,9 +4881,12 @@ def _forward_step_inter_pp(self,
 
     def _validate_token_id_range(self, request: LlmRequest) -> None:
         if isinstance(self.model_engine.model, DecoderModelForCausalLM):
-            # Only skip token‐range checks for Llama4 when the request has multimodal data
-            if isinstance(self.model_engine.model,
-                          Llama4ForConditionalGeneration):
+            # Only skip token‐range checks for Llama4 when the request has multimodal data.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `‐` (HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)